src/app/components/header/header.component.html

```diff
@@ -1,17 +1,21 @@
 <div class="main-top">
-    <div class="one-block">
-        <div class="tabs">
-            <a routerLink="/channels" routerLinkActive="active" class="tabs-item">Channels <i (click)="createChannel($event)" class="fa fa-plus"></i></a>
-            <a routerLink="/campaigns" routerLinkActive="active" class="tabs-item">Campaigns <i (click)="createCampaign($event)" class="fa fa-plus"></i></a>
-            <a class="tabs-item" href="#" (click)="toggleWidget('open_about', $event)">About</a>
-            <a class="tabs-item" href="#" (click)="toggleWidget('open_contact', $event)">Contact</a>
-        </div>
-
-        <form method="post" action="/" class="search-form">
-          <div class="imu-balance">
-            <label>Balance:</label>
-            {{balance | neobalance}} $IMU
-          </div>
-        </form>
+  <div class="one-block">
+    <div class="tabs">
+      <a routerLink="/campaigns" routerLinkActive="active" class="tabs-item">Crowdfund
```

Comment:
This is unfortunately not correct. Default and first page should still be channels. David only wanted the Campaigns to be renamed to "Crowdfunding" in the header.